tests/e2e/main.go

```diff
@@ -48,6 +59,12 @@ var (
 	gaiaTag = flag.String("gaia-tag", "", "gaia tag to use - default is latest")
 )
 
+var (
+	consumerVersions VersionSet
+	providerVersions VersionSet
+	transformGenesis = flag.Bool("transform-genesis", false, "do consumer genesis transformation for newer clients. Needed when provider chain is on an older version")
```

Comment:
I think this help string needs a bit of attention. The message is not clear to me - what should I put when checking multiple provider versions, some older and some newer? is any version != latest an older version?

Reply:
agree as this a complicated subject. referencing to the release notes to clarify when this is needed.
pls note that this is subject to be changed by [#1559](https://github.com/cosmos/interchain-security/issues/1559) which should facilitate this step